code/modules/antagonists/wraith/critters/abilities/spiker.dm

```diff
@@ -102,3 +113,50 @@
 		..()
 		var/atom/movable/screen/ability/topBar/B = src.object
 		B.UpdateOverlays(image(border_icon, border_state), "mob_type")
+
+/datum/targetable/critter/spiker/shuffle
```

Comment:
Does this have any visible effect when they use it? Doesn't have to be fancy but something to indicate that they're going Speed Mode would be good

Reply:
Yep! There's an entire new sprite!
![shufflerguy](https://user-images.githubusercontent.com/36884328/202587311-145756ce-296d-4d2e-ac6f-4f9735eda468.gif)
Scrunch.

Reply:
nice